src/main/java/com/powsybl/openloadflow/sensi/SensitivityFactorReaderAdapter.java

```diff
@@ -46,50 +46,54 @@ public SensitivityFactorReaderAdapter(Network network, SensitivityFactorsProvide
 
     @Override
     public void read(Handler handler) {
+        ContingencyContext commonContingencyContext = new ContingencyContext(ContingencyContextType.ALL, null);
```

Comment:
Just as small improvement, would be better to use 3 static methods to create ContingencyContexts (`ContingencyContext.createAllContingencyContext()`, `ContingencyContext.createNoneContingencyContext()`, `ContingencyContext.createSpecificContingencyContext(String id)`)